Superlib.js

```diff
@@ -0,0 +1,633 @@
+				let key = content.substr(0, index + 1).trim();
+				let value = content.substr(index + 1).trim();
+				switch (key) {
+					case "【作 者】":
+						doPerson(item, value);
+						break;
+					case "【学位授予单位】":
+						item.university = value;
+						break;
+					case "【导师姓名】":
+						item.creators.push({
+							lastName: value,
+							creatorType: 'contributor',
+							fieldMode: 1
+						});
+						break;
+					case "【学位年度】":
+						item.date = value;
+						break;
+					case "【摘 要】":
+						item.abstractNote = value.replace(/隐藏更多/g, '');
+						break;
+					case "【关键词】":
+						doTag(item, value);
+						break;
+					case "【学位名称】":
+						item.type = value;
+						break;
+					default:
+						break;
+				}
+			}
+			break;
+		case '会议论文':
+			item.title = ZU.trimInternal(text(doc, 'h1.title'));
+			infos = doc.querySelectorAll('#m_top li');
+			for (var section3 of infos) {
+				let content = ZU.trimInternal(section3.textContent);
+				if (!content || content.length <= 0) continue;
+		
+				let index = content.indexOf('】');
+				if (index <= -1) continue;
+		
+				content = trim(content);
+				let key = content.substr(0, index + 1).trim();
+				let value = content.substr(index + 1).trim();
+				switch (key) {
+					case "【作 者】":
+						doPerson(item, value);
+						break;
+					case "【会议名称】":
+						item.conferenceName = value;
+						break;
+					case "【会议录名称】":
+						item.series = value;
+						break;
+					case "【日 期】":
+						item.date = value;
+						break;
+					case "【摘 要】":
+						item.abstractNote = value.replace(/隐藏更多/g, '');
+						break;
+					case "【关键词】":
+						doTag(item, value);
+						break;
+					case "【作者联系方式】":
+						item.place = value;
+						break;
+					default:
+						break;
+				}
+			}
+			break;
+		case '专利':
+			item.title = ZU.trimInternal(text(doc, 'h1.title'));
+			infos = doc.querySelectorAll('.content li');
+			for (var section4 of infos) {
+				let content = ZU.trimInternal(section4.textContent);
+				if (!content || content.length <= 0) continue;
+		
+				let index = content.indexOf('】');
+				if (index <= -1) continue;
+		
+				content = trim(content);
+				let key = content.substr(0, index + 1).trim();
+				let value = content.substr(index + 1).trim();
+				switch (key) {
+					case "【申请号】":
+						item.applicationNumber = value;
+						break;
+					case "【申请人】":
+						doInventor(item, value);
+						break;
+					case "【发明人】":
+						doInventor(item, value);
+						break;
+					case "【申请日期】":
+						item.filingDate = value;
+						break;
+					case "【地 址】":
+						item.place = value;
+						break;
+					case "【专利类型】":
+						item.extra = '专利类型: ' + value;
+						break;
+					case "【IPC号】":
+						item.extra += '; IPC号: ' + value;
+						break;
+					case "【摘 要】":
+						item.abstractNote = value;
+						break;
+					default:
+						break;
+				}
+			}
+			break;
+	}
+	
+	if (item.title.startsWith('《') && item.title.endsWith('》')) {
+		item.title = item.title.replace(/《|》/g, '');
```

Comment:
same suggestion as above with this replacement (could some of this duplicated processing logic get spun out into a separate function?)